shell/src/main/java/alluxio/shell/command/CpCommand.java

```diff
@@ -555,7 +555,8 @@ private void copyFileToLocal(AlluxioURI srcPath, AlluxioURI dstPath)
     File dstFile = new File(dstPath.getPath());
     String randomSuffix =
         String.format(".%s_copyToLocal_", RandomStringUtils.randomAlphanumeric(8));
-    File tmpDst = new File(Name.HOME + randomSuffix);
+    File tmpDst = new File(PathUtils.concatPath( dstFile.getAbsolutePath(),
```

Comment:
I think we can break fixing `copyToLocal` into two parts:
1. Make it possible to copy a file into a directory by setting the directory as the destination, e.g. `bin/alluxio fs copyToLocal /LICENSE /home/username`.
2. Make it possible to use relative paths, e.g. `bin/alluxio fs copyToLocal /LICENSE dir` or `bin/alluxio fs copyToLocal /LICENSE .`

For this PR I'd like to focus on (1) and leave (2) for a second PR. When we do address (2), I think we can do it by changing `AlluxioURI` parsing so that when the scheme is `file://` it converts relative paths to absolute paths.